pyro_risks/platform_fwi/get_fwi_effis_score.py

```diff
@@ -0,0 +1,47 @@
+# Usual Imports
+from dotenv import load_dotenv
+import os
+from datetime import date
+
+# Geographic librairies Imports
+import geopandas as gpd
+from shapely.geometry import Point, Polygon
+
+# Pyro Risks Imports
+from pyro_risks.utils.s3 import S3Bucket
+
+
```

Comment:
[MEDIUM] a good practice is to state function that should be used from the script. 

```suggestion
__all__ = ["get_score"]
```